x-pack/plugins/ml/server/models/data_recognizer/modules/sample_data_ecommerce/ml/high_sum_total_sales.json

```diff
@@ -0,0 +1,38 @@
+{
+  "groups": [],
+  "description": "Kibana sample eCommerce data : detect anomalies in total sales",
+  "analysis_config" : {
+    "bucket_span": "1h",
+    "detectors": [
+      {
+        "detector_description": "Mean total sales",
+        "function": "high_sum",
+	"field_name": "taxful_total_price",
```

Comment:
There is a tab character here rather than spaces, same with the line below.
There are also other lines in other files which are using tab characters rather than spaces.
Your editor may have a setting to use spaces rather than tabs for indentation.

Reply:
those naughty tab characters are still here.
if using `prettier` from the terminal, you can recursively check each file using `/**/*.json`
e.g.
```
./node_modules/prettier/bin-prettier.js -l x-pack/plugins/ml/server/models/data_recognizer/modules/sample_*/**/*.json 
```

Reply:
Thank you!